libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -590,7 +606,8 @@ AP_GPS_UBLOX::_parse_gps(void)
     if (_class == CLASS_ACK) {
         Debug("ACK %u", (unsigned)_msg_id);
 
-        if(_msg_id == MSG_ACK_ACK) {
+        // a nack has the same effect as an ack
+        if(_msg_id == MSG_ACK_ACK || _msg_id == MSG_ACK_NACK) {
```

Comment:
I just updated the pull request to address the reason I added that line, it was around sbas config.